tests/unit/callbacks/test_console_print_span.py

```diff
@@ -75,10 +77,13 @@ def test_get_output_panel(
             json_mock.assert_called_once()
             panel_mock.assert_called_once()
 
-    # Skip frameworks that end with a tool call
+    # Skip frameworks that end with a tool call or now return JSON in final LLM call
     if request.node.callspec.id not in (
```

Comment:
I believe this list now contains all the frameworks, so we could remove this check?